src/main/webapp/js/grapheditor/EditorUi.js

```diff
@@ -1514,6 +1514,7 @@ EditorUi.prototype.installShapePicker = function()
 	
 	graph.dblClick = function(evt, cell)
 	{
+		return;
```

Comment:
I think we can remove the code below

Reply:
I just disabled the **dblClick**, 
the code below can be useful in the futur.